src/pages/CatIndex.js

```diff
@@ -1,8 +1,27 @@
 import React from "react"
-
-const Catindex= () => {
+import { NavLink as RouterNavLink } from "react-router-dom"
+const Catindex= ({cats}) => {
+    
 return (
-    <h1>Cat Index</h1>
+    <main>
+        {cats?.map((cat) => {
+            return (
+                <>
+                <div key={cat.id}>
+                    <img
+                    src={cat.image}
+                    alt="profile of all our cats friends"
+                    className="cat-profile-pic"
+                    />
+                    <RouterNavLink to= {`/catshow/${cat.id}`}>
+                     <p>{cat.name}</p>
+                     <p>{cat.age}</p>
+                    </RouterNavLink>
+                </div>
```

Comment:
This should be indented one more tab since it is nested inside the React fragments.